ui/pages/confirmations/utils/confirm.ts

```diff
@@ -23,3 +25,9 @@ const SIGNATURE_APPROVAL_TYPES = [
 export const isSignatureApprovalRequest = (
   request: ApprovalRequest<Record<string, Json>>,
 ) => SIGNATURE_APPROVAL_TYPES.includes(request.type as ApprovalType);
+
+export const parseTypedDataMessage = (dataToParse: string) => {
+  const { message, domain = {}, primaryType, types } = JSON.parse(dataToParse);
+  const sanitizedMessage = sanitizeMessage(message, primaryType, types);
+  return { domain, sanitizedMessage, primaryType };
```

Comment:
Q: was adding domain in the return here deliberate?

Reply:
Yep its used in code calling it.

Reply:
sorry, where exactly is this?

Reply:
👍 I updated the PR to address this